Client/Frontend/Browser/BrowserViewController/BrowserViewController+WebViewDelegates.swift

```diff
@@ -187,6 +187,14 @@ extension BrowserViewController: WKNavigationDelegate {
             return
         }
 
+        if url.scheme == "javascript", navigationAction.request.isPrivileged {
```

Comment:
Is this from Justin's patch. Was it merged on to your branch rather than rebased?

Reply:
i did a rebase, not sure how this appeared, crazy!

Reply:
I fixed up the rebase on the patch, is ok now.